LDM-592.tex

```diff
@@ -954,6 +954,31 @@ \section{Data Release Processing}
 In some contexts this may not remove all record of these datasets held outside their primary storage locations, as long as the storage cost of those records is negligible.
 }
 
+\usecase{DRP31}{Compare with previous data releases}{%
+
+\item
+A new data release is prepared by the Production Operators.
+
+\item
+The Validation Scientist analyzes a subset of the new data release by directly comparing it with the current Data Release.
+
+\item
+A statistical analysis is done with the catalogs derived from both data releases to determine if the new data release is suitable for release.
+
+}
+
+\usecase{DRP32}{Use of rerun collections}{%
+
+\item
+A Developer runs some processing starting from outputs produced by another Developer, where those outputs are specified as inputs using a label given as a single string (possibly with path-like namespace components).
```

Comment:
After much discussion with Tim, I think that my problem (as regards use cases) stems from this sentence.  If it said "A Developer runs some processing starting from outputs produced by the Production Operators or another Developer" I'd be happy enough.

Separately, we should modify the developer guide to say that the "rerun" mechanism should be used instead of explicitly specifying input and output directories.